kvmagent/kvmagent/plugins/imagestore.py

```
@@ -136,6 +136,48 @@ def _get_progress(synced):
+            return infos
+
+
+        volumes = ""
+        for volume_info in volume_infos:
+            target_disk, _ = vm._get_target_disk(volume_info.volume)
+            node_name = self.get_disk_device_name(target_disk)
+
+            target_install_path = volume_info.target
+            volumes += ",".join([node_name, target_install_path]) + ";"
+
+        PFILE = linux.create_temp_file()
```

Ruff (0.8.2)

`linux` may be undefined, or defined from star imports

(F405)